src/main/scala/where/Condition.scala

```diff
@@ -76,3 +77,12 @@ case class Expression(string: String, references: Reference*) extends Condition
   def identifiables: Set[Identifiable] =
     (references flatMap (_.getReferenceable)).toSet
 }
+
+case class NoRelationshipCondition(node: Node) extends Condition {
```

Comment:
i'm thinking about the naming here. i think from a usage perspective, i'd prefer:

```scala
val query = node where HasNoRelationships(node) returns startNode
```
vs
```scala
val query = node where NoRelationshipCondition(node) returns startNode
```
what do you think? also @a-wortman ?